src/types/checkbox.js

```diff
@@ -20,6 +21,8 @@ export default multiple({
                     value   : opt.value,
                     checked : opt.selected,
 
+                    required : req,
```

Comment:
I tried to use `options.required` everywhere initially. For some reason I didn't think I had access to `options` inside these `map()` functions (as in, I thought I remembered trying to use `options.required` and getting an error. Oh well, fixing now.